src/sections/Projects/Sistent/sistent.style.js

```diff
@@ -687,6 +688,17 @@ const SistentWrapper = styled.div`
     color: ${(props) => props.theme.whiteToBlack};
   }
 
+  .card {
```

Comment:
since its the same styles we are following from blog section. So please try to reuse the classes rather than redefining everytime.

Reply:
Yes @sudhanshutech learned few things from this pr, will implement it.